relevanceai/operations/cluster/cluster.py

```diff
@@ -363,6 +362,26 @@ def _get_document_vector_field(self):
         vector_fields.append(most_recent_updated_vector)
         return vector_fields
 
+    def _retrieve_dataset_id(self, dataset) -> str:
+        """Helper method to get multiple dataset values.
+        Dataset can be either a Dataset or a string object."""
+        from relevanceai.dataset import Dataset
```

Comment:
Have you tested this import here? This might cause `ImportError` due to recursive import.

Reply:
Idk this code was originally in here.